File: .NET/Backend/project_vc#/project_vc#/DTOs/DefaultConfigurationDTO.cs
namespace project_vc_.DTOs;

public class DefaultConfigurationDTO
{
    public int Id { get; set; }
    public string? Name { get; set; }
    public string? CompType { get; set; }

    public DefaultConfigurationDTO() { }
    public DefaultConfigurationDTO(int id, string? name, string? compType)
    {
        Id = id;
        Name = name;
        CompType = compType;
    }
}

File: Spring Boot/Backend/v-conf/src/main/java/com/example/dto/DefaultConfigurationDTO.java
package com.example.dto;

public class DefaultConfigurationDTO {

    private Integer id;
    private String name;
    private String compType;   // S / I / E / C

    public DefaultConfigurationDTO(
            Integer id,
            String name,
            String compType
    ) {
        this.id = id;
        this.name = name;
        this.compType = compType;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getCompType() {
        return compType;
    }
}
